packages/ordered-list/src/ListItem/ListItem.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { useDescendant } from '@leafygreen-ui/descendants';
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { Body, Description } from '@leafygreen-ui/typography';
+
+import { OrderedListContext } from '../OrderedListContext/OrderedListContext';
+
+import {
+  baseStyles,
+  contentClassName,
+  contentStyles,
+  getThemedStateStyles,
+  stepIconClassName,
+  stepStyles,
+  titleStyles,
+} from './ListItem.styles';
+import { ListItemProps } from './ListItem.types';
+
+const ListItem = React.forwardRef(
```

Comment:
Nit: this is probably fine since it's exported from the ordered list package, but this component bakes in the numbering, which makes me think calling this `OrderedListItem` might be more accurate.

Reply:
i thought it might be too verbose, but i can update if you think that's better

Reply:
I can imagine a future where we have an unordered list component, and if someone wants to use both on the same page, they'd have to rename the component import themselves. IMO being a little more explicit here it's probably better.

Reply:
kk will update